File: testing/manual_benchmarking/Python/prime_bug3.py
# prime_bug3.py

def is_prime(n):
    if n <= 1:
        return False

    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False

    return True


num = int(input("Enter a number: "))

if is_prim(num):
    print(f"{num} is prime")
else:
    print(f"{num} is not prime")

File: testing/manual_benchmarking/JavaScript/prime_bug3.js
// prime_bug3.js

function isPrime(n) {
    if (n <= 1) return false;

    for (let i = 2; i * i <= n; i++) {
        if (n % i === 0)
            return false;
    }
    return true;
}

let num = 5;

if (isPrim(num))
    console.log(num + " is prime");
else
    console.log(num + " is not prime");